<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="ATMECSINLT-060" ignored="0" name="SampleTest" tests="1" failures="1" timestamp="25 Feb 2020 10:57:58 GMT" time="22.852" errors="0">
  <testcase name="totalTestCases" time="22.329" classname="com.atmecs.qa.falcondashboard.testscript.TC27_TotalTestCases">
    <failure type="java.lang.AssertionError" message="expected [RUN IN - 8 secs] but found [RUN IN - 9 secs]">
      <![CDATA[java.lang.AssertionError: expected [RUN IN - 8 secs] but found [RUN IN - 9 secs]
at com.atmecs.qa.falcondashboard.helper.ValidationHelper.getDataofRow(ValidationHelper.java:29)
at com.atmecs.qa.falcondashboard.validationresults.ViewPageValidation.validateTime(ViewPageValidation.java:28)
at com.atmecs.qa.falcondashboard.testscript.TC27_TotalTestCases.totalTestCases(TC27_TotalTestCases.java:52)
at com.atmecs.qa.falcondashboard.runner.TestNGRunner.main(TestNGRunner.java:59)
at org.codehaus.mojo.exec.ExecJavaMojo$1.run(ExecJavaMojo.java:294)
at java.lang.Thread.run(Thread.java:748)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- totalTestCases -->
</testsuite> <!-- SampleTest -->
